AcceptanceTest.md

```diff
@@ -0,0 +1,23 @@
+# AcceptanceTest for QualityToolsExtension.jar
+
+Due to the difficulty of JUnit tests developers created the following acceptance test to verify the start up functionality of the BlueJ application. 
```

Comment:
What is meant by "Due to the difficulty of JUnit tests"?  What is the intent of the opening phrase of this document?

Reply:
Invoking the BlueJ proxy object to create a JUnit test.